cmd/keymasterd/main.go

```diff
@@ -1614,12 +1614,37 @@ func (state *RuntimeState) u2fSignResponse(w http.ResponseWriter, r *http.Reques
 	http.Error(w, "error verifying response", http.StatusInternalServerError)
 }
 
+func (state *RuntimeState) IsAdminUser(user string) bool {
+	for _, adminUser := range state.Config.Base.Admins {
+		if user == adminUser {
+			return true
+		}
+	}
+	return false
+}
+
 const profilePath = "/profile/"
 
+func profileURI(authUser, effectiveUser string) string {
```

Comment:
effectiveUser is confuring. AssumedUser is better. Otherwise LGTM, also do you plan to add coverage to this?